### DISCUSSION TITLE
Page Not Found

### DISCUSSION THREAD

**Discussion by chayou20** [naver/egjs-flicking#750] [Q&A]
When I changed homepage language to Korean, 'Page Not Found' page is showed in local homepage.
I don't know why this error is showed.

and I got this error "Node Sass does not yet support your current environment: Windows 64-bit with Unsupported runtime" when I used node.js version 19.

and I read this page "https://github.com/sass/node-sass/releases/tag/v4.14.1"

node-sass used in Winodws 64-bit not work in node.js version 19.
only work in node.js version 0.10 to 14

Could I add this to docs readme.md

**Comment by WoodNeck** [comment]
PRs are welcome :)

You can try these for specifying the supported node.js versions:
- https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json
- https://shields.io/category/platform-support (for showing badge)

**Comment by malangfox** [comment]
Hello @chayou20.

[Each locale of Docusaurus is a distinct standalone single-page application.](https://docusaurus.io/docs/i18n/tutorial)
You cannot start multiple locale Docusaurus sites at the same time, but you can test Korean pages in your local environment by running the command below at docs folder.

`npm run start -- --locale ko`

If this command doesn't work for your environment, please leave additional comments.
